com.github.PopoutApps.popout3d.yml

```diff
@@ -0,0 +1,173 @@
+app-id: com.github.PopoutApps.popout3d
+
+runtime: org.freedesktop.Sdk
```

Comment:
This should be `org.freedesktop.Platform`

Reply:
Done.